linux/main.go

```diff
@@ -116,3 +129,21 @@ document.addEventListener('click', function (e) {
 	w.Run()
 	server.Shutdown()
 }
+
+
+func getStore(config *config.Configuration, logger *mlog.Logger) (store.Store, error) {
+	var db store.Store
+	var err error
+	db, err = sqlstore.New(config.DBType, config.DBConfigString, config.DBTablePrefix, logger, nil)
+	if err != nil {
+		return nil, err
+	}
+	if config.AuthMode == server.MattermostAuthMod {
+		layeredStore, err2 := mattermostauthlayer.New(config.DBType, config.DBConfigString, db)
```

Comment:
If you haven't modify the mattermostauthlayer, it is going to create another connection to the database, so probably instead of passing the DBType and the DBConfigString, it should pass the connection already in use. (This is this way because you can use different databases, in theory to have mattermost and focalboard in personal server connected, but that is a corner case).

Reply:
Ah interesting! Wasn't aware of that. Fixing.